### ISSUE TITLE
No Label Text in ThemeSwapper

### ISSUE THREAD

**Issue by timrlai** [timrlai/timrlai#89]
### Description of Task

There is an accessibility error because there is no text in the label element in the ThemeSwapper component. Even though there is an aria-labelledby attribute on the input, the label element should either include visually hidden text or should be changed to a div or something neutral.
